packages/asset-kit-block/src/components/AssetCount.tsx

```diff
@@ -16,5 +16,5 @@ export const AssetCount = ({ count, color }: AssetCountProps): ReactElement => {
         );
     }
 
-    return <div className="tw-text-black-50">Add assets to make them available</div>;
+    return <div className="tw-text-black-50 tw-opacity-70">Add assets to make them available</div>;
```

Comment:
You also need to change the color from black-50, to the body color defined in design settings.

To do so you can use

```
style={{
    color: BlockStyles[TextStyles.p].color,
}}
```

You can import these from 
`@frontify/guideline-blocks-settings`